spec/rails_helper.rb

```diff
@@ -0,0 +1,63 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```

Comment:
This is a default file created by `rails generate rspec:install` (see the [RSpec Rails docs](https://relishapp.com/rspec/rspec-rails/docs))